app/models/pacbio/well.rb

```diff
@@ -111,7 +85,7 @@ def summary
 
     # A collection of all the used_aliquots for given libraries and pools in a well
     def base_used_aliquots
-      used_aliquots.collect(&:source).collect(&:used_aliquots).flatten
+      used_aliquots.collect(&:source).collect(&:used_aliquots).flatten.uniq(&:id)
```

Comment:
This is a bit of a code smell. They should always be unique.

Reply:
Yes, this actually goes along with the issue we talked about where the validation is happening before the update. What it appears to me while debugging is that when we update an aliquot in a well using used_aliquots_attributes, for example, a well with two aliquots and we update one of those, it validates the well and it has three aliquots (two of them are the same, where one is the original object and one is the object to be updated).